src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpClientIntegration.cs

```diff
@@ -0,0 +1,772 @@
+            int opCode,
+            int mdToken,
+            long moduleVersionPtr)
+        {
+            if (handler == null)
+            {
+                throw new ArgumentNullException(nameof(handler));
+            }
+
+            var callOpCode = (OpCodeValue)opCode;
+            var httpClient = handler.GetInstrumentedType(HttpClientTarget);
+            var cancellationToken = (CancellationToken)boxedCancellationToken;
+
+            Func<HttpClient, Uri, CancellationToken, Task<HttpResponseMessage>> instrumentedMethod = null;
+
+            try
+            {
+                instrumentedMethod =
+                    MethodBuilder<Func<HttpClient, Uri, CancellationToken, Task<HttpResponseMessage>>>
+                       .Start(moduleVersionPtr, mdToken, opCode, DeleteAsync)
+                       .WithConcreteType(httpClient)
+                       .WithParameters(uri, cancellationToken)
+                       .WithNamespaceAndNameFilters(ClrNames.GenericTask, ClrNames.Uri, ClrNames.CancellationToken)
+                       .Build();
+            }
+            catch (Exception ex)
+            {
+                Log.ErrorRetrievingMethod(
+                    exception: ex,
+                    moduleVersionPointer: moduleVersionPtr,
+                    mdToken: mdToken,
+                    opCode: opCode,
+                    instrumentedType: HttpClientTarget,
+                    methodName: DeleteAsync,
+                    instanceType: handler.GetType().AssemblyQualifiedName);
+                throw;
+            }
+
+            return DeleteAsyncInternal(
+                instrumentedMethod,
+                reportedType: callOpCode == OpCodeValue.Call ? httpClient : handler.GetType(),
+                (HttpClient)handler,
+                (Uri)uri,
+                cancellationToken);
+        }
+
+        private static async Task<HttpResponseMessage> DeleteAsyncInternal(
+            Func<HttpClient, Uri, CancellationToken, Task<HttpResponseMessage>> deleteAsync,
+            Type reportedType,
+            HttpClient handler,
+            Uri uri,
+            CancellationToken cancellationToken)
+        {
+            using (var scope = ScopeFactory.CreateOutboundHttpScope(Tracer.Instance, "DELETE", uri, IntegrationName))
+            {
+                try
+                {
+                    HttpResponseMessage response = await deleteAsync(handler, uri, cancellationToken).ConfigureAwait(false);
+
+                    // this tag can only be set after the response is returned
+                    scope?.Span.SetTag(Tags.HttpStatusCode, ((int)response.StatusCode).ToString());
+
+                    return response;
+                }
+                catch (Exception ex)
+                {
+                    scope?.Span.SetException(ex);
+                    throw;
+                }
+            }
+        }
+
+#pragma warning disable CS0419 // Ambiguous reference in cref attribute
```

Comment:
What "ambiguous reference" is this referring to? Is it something we can fix instead of ignoring the warning?

Reply:
Fixing this by adding the parameter types to the xml comment.